exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -97,6 +98,12 @@ public void Roll(int roll)
             }
         }
 
+        private void PenaltyBoxMessage(bool gettingOut = false)
+        {
+            Console.WriteLine(
+                $"{players[currentPlayer]} {(gettingOut ? "is" : "is not")} getting out of the penalty box");
+        }
```

Comment:
I really like the use of comparing 'is' and 'isn't' in the message string. Very tidy code!